packages/a11y/package.json

```diff
@@ -28,6 +28,7 @@
 	"module": "build-module/index.js",
 	"react-native": "src/index",
 	"types": "build-types",
+	"wpScriptModuleExports": "./build-module/module/index.js",
```

Comment:
Could we use in this packages the `.mjs` extension to avoid this `build-module/module` path? Would it cause some issues?

"wpScriptModuleExports": "./build-module/index.mjs",

Reply:
Sure, we could try that. It would have a similar effect on the file as using `type: module` would on the entire package.

I'll give that a try, it may be a good way to signal in this package that the file is a module. Of course everything in build-module should be "modules" 🤔

Reply:
Doable, but kind of involved. Let's leave this to consider in a follow-up: https://github.com/WordPress/gutenberg/pull/65271

Reply:
Sure thing 😄